tailwind.config.js

```diff
@@ -19,7 +19,12 @@ module.exports = {
         hotPink: '#FF1966',
         dark: '#111111',
         light: '#FAFAFA',
-        violetDark: '#4c2889'
+        violetDark: '#4c2889',
+        'dark-gray-4': '#2E2E2E',
+        'dark-gray-7': '#8f8f8f',
+        'base-gray-900': '#666666',
+        'light-gray-4': '#EAEAEA',
+        'vercel-blue': '#0C4BE6'
```

Comment:
IDK how I feel about this... I'd *prefer* if we could just stick to consistent out of the box Tailwind colors.